omod/src/main/resources/config.xml

```diff
@@ -13,6 +13,7 @@
 	<activator>@MODULE_PACKAGE@.IdgenModuleActivator</activator>
 	<require_modules>
 		<require_module>org.openmrs.module.webservices.rest</require_module>
+		<require_module version="2.0.1">org.openmrs.module.idgen</require_module>
```

Comment:
The idgen module cannot require itself. You can read more about that from: https://wiki.openmrs.org/display/docs/Module+Config+File

Reply:
Fixed. The <require_module> tag is for external modules needed in the current module.